web/client.js

```diff
@@ -595,14 +595,52 @@ function canTarget(playerIdx) {
     // Cannot target dead player.
     return player.influenceCount() > 0;
 }
+
+var doublekillAction;
+var doubleKillWarningDisabled;
+function confirmDoubleKillAction() {
+    $('#doubleRevealWarning').modal('hide');
+    if (doublekillAction) {
+        doublekillAction();
+    }
+}
+function disableDoubleKillWarning() {
+    doubleKillWarningDisabled = true;
```

Comment:
It would be nice to store this in local storage. There are already functions to do this: `localStorageGet` and `localStorageSet`